src/Couchbase.Lite.Shared/Support/UWP/Reachability.cs

```diff
@@ -65,6 +65,11 @@ public void Stop()
         {
             NetworkInformation.NetworkStatusChanged -= OnNetworkStatusChanged;
         }
+        
+        public void Dispose()
+        {
+            Stop();
+        }
```

Comment:
Originally I want to set the Url to null in Dispose....

Reply:
Remove Dispose method, just simply call Stop() instead since url needs to be existed as long as WebSocketWrapper/Replicator stays alive...